docs/reference/modules/indices/recovery.asciidoc

```diff
@@ -48,3 +48,11 @@ sent in parallel for each recovery. Defaults to `2`.
 +
 You can increase the value of this setting when the recovery of a single shard
 is not reaching the traffic limit set by `indices.recovery.max_bytes_per_sec`.
+
+`indices.recovery.max_concurrent_operation_chunks`::
```

Comment:
Suggest simply `indices.recovery.max_concurrent_operations`; the fact that operations are sent in chunks (i.e. batches) is a bit too much detail for users here IMO.

Reply:
++, see https://github.com/elastic/elasticsearch/pull/58018/commits/e4f1fbfed406b007d3cc24b18781ce1b36607120.